data_steward/tools/generate_ext_tables.py

```diff
@@ -66,12 +72,13 @@ def get_mapping_table_ids(project_id, mapping_dataset_id):
     :param mapping_dataset_id: dataset_id containing mapping tables
     :return: returns mapping table ids
     """
-    table_objs = bq_utils.list_tables(mapping_dataset_id, project_id=project_id)
+    client = bq.get_client(project_id)
+    dataset_ref = DatasetReference(project_id, mapping_dataset_id)
+    table_objs = bq.list_tables(client, dataset_ref)
     mapping_table_ids = []
     for table_obj in table_objs:
-        table_id = bq_utils.get_table_id_from_obj(table_obj)
-        if table_id in MAPPING_TABLES:
-            mapping_table_ids.append(table_id)
+        if table_obj.table_id in MAPPING_TABLES:
+            mapping_table_ids.append(table_obj.table_id)
```

Comment:
This can use list comprehension:
```
    mapping_table_ids = [table_obj.table_id for table_obj in table_objs if table_obj.table_id in MAPPING_TABLES]
```

Reply:
I like the list comprehension better. I will make this change!